.github/workflows/build.yaml

```diff
@@ -163,6 +163,38 @@ jobs:
           BOT: ${{ matrix.bot }}
           DEVICE: ${{ matrix.device }}
           SHARD: ${{ matrix.shard }}
+          DEVTOOLS_PACKAGE: devtools_app
+        run: ./tool/bots.sh
+
+      - name: Upload Golden Failure Artifacts
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32
+        if: failure()
+        with:
+          name: golden_image_failures.${{ matrix.bot }}
+          path: packages/devtools_app/integration_test/**/failures/*.png
+
+  devtools-extensions-integration-test:
+    name: devtools_extensions integration-test ${{ matrix.bot }}
+    needs: flutter-prep
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # Consider running integration tests in ddc mode, too.
+        bot: [integration_dart2js]
+    steps:
+      - name: git clone
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
+      - name: Load Cached Flutter SDK
+        uses: actions/cache@704facf57e6136b1bc63b828d79edcd491f0ee84
+        with:
+          path: |
+            ./flutter-sdk
+          key: flutter-sdk-${{ runner.os }}-${{ needs.flutter-prep.outputs.latest_flutter_candidate }}
+      - name: tool/bots.sh
+        env:
+          BOT: ${{ matrix.bot }}
+          DEVTOOLS_PACKAGE: devtools_extensions
```

Comment:
do the golden artifacts, below for this workflow, need to point to devtools_extensions/ instead of devtools_app/?

Reply:
we don't need these at all actually, removed this step